### ISSUE TITLE
WillPopScope functionality 

### ISSUE THREAD

**Issue by morad00** [apptreesoftware/flutter_google_map_view#107]
Is there any way to handle back button (in android) after opening the map? I saw a widget called seems that's what I need but how to use it here ? 
I want to attempt the user to dismiss the map_view

**Comment by MathieuAuclair** [comment]
Are you trying to handle the close button in the scaffold?

**Comment by morad00** [comment]
@MathieuAuclair something like this but inside the `mapView.show()` not for the main scaffold

**Comment by MathieuAuclair** [comment]
Yes but when you execute ```mapView.show()```, you get a new widget with it's own scaffold, are you trying to control the close button?
![debug](https://user-images.githubusercontent.com/22243720/44998220-cf65d700-af81-11e8-809e-91faeb94bd40.png)

**Comment by edufolly** [comment]
Maybe a stream to send for the main class a information that the map was close.
